webpack/stories/docs/api-middleware.stories.mdx

```diff
@@ -150,3 +150,23 @@ dispatch(
   })
 );
 ```
+
+## Update resources object
+
+There are some cases in which the resources list, that fetched earlier, needs to be altered (update, delete), instead of making two separated API calls (for altering and fetching the up-to-date resources list),
+the API middleware can accept an `updatedData` object, with the updated resource object corresponding to its key. 
+> ⚠️ **Warning**: Please be aware that the given `updatedData` should be corresponding with the API call.
+
+Example:
+
+```js
+// This action creates a post API call and only if it ends succesfully,
+// the response object will update with the given 'updatedData' object
+dispatch(
+  post({
+    key: MY_SPECIAL_KEY,
+    url,
+    updatedData: {results: ['altered resource']}
```

Comment:
What format must this data be in?  I'm guessing `updatedData` wants a data shape identical to the entire response?  Maybe it's good to call that out above.

Reply:
not the all response object, but only the `data` part of it,
it  depends on the data structure of each API call, mostly it is something like 
```js
{ results: [{id: 1 , name: 'A'}, {id: 2 , name: 'B'}] } 
```
But this structure might be different.